pylint/checkers/typecheck.py

```diff
@@ -399,6 +399,14 @@ def _missing_member_hint(
         "isinstance-second-argument-not-valid-type",
         "Emitted when the second argument of an isinstance call is not a type.",
     ),
+    "W1117": (
+        "The keyword argument %r will be added to the keyword variadic "
+        "parameter dictionary %r since it has the same name as a positional-only parameter",
+        "hidden-kwarg",
```

Comment:
```suggestion
        "duplicated-arg-hidden-in-kwarg",
```

```suggestion
        "duplicate-keyword-argument",
```

```suggestion
        "kwarg-hidden-by-posarg",
```

I'm trying to come up with something more descriptive. We have https://pylint.pycqa.org/en/latest/user_guide/messages/error/duplicate-argument-name.html that is close.

Reply:
Thanks for thinking along on this @Pierre-Sassoulas. I found it difficult to think of an accurate name :D.

- I think `hidden` may be a bit confusing as a term.
- Regarding `duplicate-keyword-argument`; it suggests to me 2 or more keywords were passed to the function call explicitly which isn't the case - also the message is specifically emitted when positional-only parameters are present, so I thought it would be good to make that explicit.

What do you think of this one:
`same-keyword-as-positional-with-kwargs` (inspired by the name of a [unittest in cpython](https://github.com/python/cpython/blob/main/Lib/test/test_positional_only_arg.py#LL326C21-L326C21))
Bit of a mouthful but I like that it uses the words _positional_ and _kwargs_ which are required for the message to be emitted.

Reply:
Maybe "kwarg-overridden-by-positional-arg", "kwarg-superseded-by-positional-arg" ?

Reply:
I'd say the latter suggestion, `kwarg-superseded-by-positional-arg`, is more accurate; let's try it out.